GYBITGTests/VideoRepositoryTests.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  VideoRepositoryTests.swift
+//  GYBITGTests
+//
+//  Created by juanita aguilar on 5/9/19.
+//
+
+import XCTest
+@testable import GYBITG
+import CoreMedia
+
+class VideoRepositoryTests: XCTestCase {
+var sut: VideoRepository!
```

Comment:
Create a protocol for the video repository so that you can use different versions of the repo